### PR TITLE
chore: update docs with master/main filter in workflow

### PR DESCRIPTION
**Pull request by ivo-toby**

### CODE REVIEW

**Review by andipaetzold — APPROVED** [review]

**Review by vikaskumr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ jobs:
   deploy_job:
     runs-on: ubuntu-latest
     name: Deploy app to Contentful
+    if: contains(github.ref, 'main') || contains(github.ref, 'master')
     steps:
       - uses: actions/checkout@v3
       - name: Use Node.js 16.x
```
Comment (vikaskumr) [inline README.md:39]: Should we change the to `Node.js 18.x` as we have changed in changelog.

Comment (vikaskumr) [reply README.md:39]: And we do a release then version will also change i.e. `v1.0.1`, may be we need to change that version in doc and change log also 🤔

Comment (vikaskumr) [reply README.md:39]: Ohh yes make sense, understood. we can make a release version tag to different tag.

Comment (ivo-toby) [reply README.md:39]: @vikaskumr the actual workflow is running 16 too, I'd suggest changing those in both Readme and the workflow, but that's beyond the scope of this ticket. I'll create an upgrade ticket for it

**Pull request merged**